### PR TITLE
CVE-2015-6768 and CVE-2016-5156

### PR DESCRIPTION
**Pull request by KyleRohde**
Updated docs for SWEN-331 history assignment.

### CODE REVIEW

File: cves/CVE-2015-6768.yml [hunk 1]
```diff
@@ -32,7 +32,12 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: When stopping an HTML frame and its children, there is a point where
```
Comment (mesternefeld) [inline cves/CVE-2015-6768.yml]: It seems as though your build is failing. Multi-line answers should be like my suggestion below. it should be like this for any multi-line answers in both your files. Please fix them and make sure the build passes. For another example, look how the multi-line instructions handle it.
```suggestion
description: |
  When stopping an HTML frame and its children, there is a point where
```

File: cves/CVE-2016-5156.yml [hunk 2]
```diff
@@ -198,4 +213,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: |
+    There was a coding mistake made because it was assumed event woulds
```
Comment (hollyhastings) [inline cves/CVE-2016-5156.yml]: This is very vague. What does this vulnerability imply about the team/situation during this time. What can we learn from this?

Also change "woulds" to "would"

File: cves/CVE-2015-6768.yml [hunk 3]
```diff
@@ -4,14 +4,14 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 
+CWE: CWE-703 Improper Check or Handling of Exceptional Conditions
```
Comment (scioffi) [inline cves/CVE-2015-6768.yml]: The CWE value should just be the CWE ID number.
```suggestion
CWE: 703
```

File: cves/CVE-2016-5156.yml [hunk 4]
```diff
@@ -4,14 +4,14 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 
+CWE: CWE-703 Improper Check or Handling of Exceptional Conditions
```
[the remark this answers is not in the document]
Comment (scioffi) [reply cves/CVE-2016-5156.yml]: Whichever CWE you decide to use, be sure to only use the CWE's ID number.

**Pull request merged**